sdk/documentintelligence/azure-ai-documentintelligence/README.md

```diff
@@ -250,6 +253,19 @@ for page in result.pages:
                 f"'{selection_mark.polygon}' and has a confidence of {selection_mark.confidence}"
             )
 
+if result.paragraphs:
+    print(f"----Detected #{len(result.paragraphs)} paragraphs in the document----")
+    for paragraph in result.paragraphs:
+        print(f"Found paragraph with role: '{paragraph.role}' within {paragraph.bounding_regions} bounding region")
```

Comment:
Does bounding_regions have a reasonable string representation?

Reply:
No, added a helper function to print the bouding_regions in a reasonable string.